docs/topics/http/middleware.txt

```diff
@@ -260,8 +260,7 @@ of caveats:
   define ``__init__`` as requiring any arguments.
 
 * Unlike the ``process_*`` methods which get called once per request,
-  ``__init__`` gets called only *once*, when the Web server responds to the
-  first request.
+  ``__init__`` gets called only *once*, when the Web server starts.
```

Comment:
Add a `.. versionchanged:: 1.10` annotation below this as described in [documenting new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features). Also, a mention in the 1.10 release notes wouldn't hurt.